guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -0,0 +1,134 @@
+----
+# subscription-manager repos --enable={RepoRHEL7Server} \
+--enable={RepoRHEL7ServerSatelliteCapsuleProductVersion} \
+--enable={RepoRHEL7ServerSatelliteMaintenanceProductVersion} \
+--enable={RepoRHEL7ServerSatelliteToolsProductVersion} \
+--enable={RepoRHEL7ServerSoftwareCollections} \
+--enable={RepoRHEL7ServerAnsible}
+----
++
+endif::[]
+
+NOTE: If you are installing {ProductName} as a virtual machine hosted on {oVirt}, you must also enable the *Red{nbsp}Hat Common* repository, and install {oVirt} guest agents and drivers. For more information, see https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/virtual_machine_management_guide/installing_guest_agents_and_drivers_linux#Installing_the_Guest_Agents_and_Drivers_on_Red_Hat_Enterprise_Linux[Installing the Guest Agents and Drivers on Red Hat Enterprise Linux] in the _Virtual Machine Management Guide_ for more information.
++
+. Clear any metadata:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-clean} all
+----
++
+. Optional: Verify that the required repositories are enabled:
++
+[options="nowrap"]
+----
+# yum repolist enabled
+----
+
+ifeval::["{build}" == "foreman"]
++
+. Install the `foreman-release.rpm` package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum localinstall https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+----
++
+. Install the `katello-repos-latest.rpm` package
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum localinstall https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+----
++
+. Install the `puppet6-release-el-7.noarch.rpm` package:
++
+----
+# yum localinstall https://yum.puppet.com/puppet6-release-el-7.noarch.rpm
+----
++
+. Install the `epel-release-latest-7.noarch.rpm` package:
++
+----
+# yum localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+----
++
+. Install the `foreman-release-scl` package:
++
+----
+# yum install foreman-release-scl
+----
+endif::[]
+
+ifeval::["{build}" == "foreman"]
+
+.CentOS Users
+If you use a CentOS operating system, complete the following steps:
+
+. Install the `foreman-release.rpm` package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum localinstall https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+----
++
+. Install the `katello-repos-latest.rpm` package
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum localinstall https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+----
++
+. Install the `puppet6-release-el-7.noarch.rpm` package:
++
+----
+# yum localinstall https://yum.puppet.com/puppet6-release-el-7.noarch.rpm
+----
++
+. Install the `epel-release-latest-7.noarch.rpm` package:
++
+----
+# yum localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
On CentOS you can simply `yum install epel-release` as long as CentOS extras is enabled

Reply:
We do not enable CentOS extras in this procedure. Is it worth enabling this CentOS extras repo to install the `epel-release` package? If this is the only task for which the CenOS extras are used, maybe it is an overkill?

Reply:
CentOS extras is enabled by default on a fresh installation. Extras just means it's not part of base RHEL.

Reply:
Thank you for taking a look at this, @ekohl. I am applying your corrections in #168.